docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -0,0 +1,90 @@
+
+Main concepts
+-------------
+
+Experiment
+^^^^^^^^^^
+
+An experiment is made up of all the different bricks needed to perform a federated learning training and testing: the training data, the algorithm used to do the local training, the federated learning strategy, the metric and the test data.
+Launching an experiment creates a :ref:`concept_compute_plan`.
+
+
+Algorithm
+^^^^^^^^^
+
+.. warning::
+    A Substrafl algorithm is not the same as a Substra :ref:`concept_algorithm`!
+
+A Substrafl algorithm contains the local training and predict code and all the associated hyper parameters (batch size, loss, optimizer, etc).
+
+
+Evaluation Strategy
+^^^^^^^^^^^^^^^^^^^
+
+The evaluation strategy specifies how and when the model is tested. More specifically it defines:
+
+* on which test data the model is tested
+* at which rounds the model is tested
+
+
+Index Generator
+^^^^^^^^^^^^^^^
+
+The notion of epochs does not fully apply to the FL setting. Usually we don't want to train on a full epoch on each organization at every round but on a lesser amount of data to prevent models from different organizations from diverging too much.
+In a federated setting, at each round, in each organization, the model is trained for ``num_updates`` batches, each batch containing ``batch_size`` data points.
+
+For instance you have a dataset of 1000 data points at every organization. You specify ``num_updates=10`` and ``batch_size=32``. At each round your model trains on 10x32=320 data points per organization.
+
+The index generator remembers which data has been used in the previous rounds and generates the new batches so that the model is trained on the full dataset (given enough number of rounds and updates). When the whole dataset has been used, the index generator shuffles the data and starts generating batches from the whole dataset again.
+
+
+Node
+^^^^
+There are three types of node:
+
+* TrainDataNode: one of the organizations the local training takes place on, with a set of data samples and an opener used for training.
+* TestDataNode: one of the organizations the model evaluation takes place on, with a set of data samples and an opener used for testing.
+* AggregationNode: the organization on which the aggregation, if there is one, takes place.
+
+
+Federated Learning Strategies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A FL strategy describes how to train a model on distributed data. The most well known strategy is the Federated Averaging strategy: train locally a model on every organization, then aggregate the weight updates from every organization, and then apply locally at each organization the averaged gradients. A strategy imposes some constraints on the model that can be used. For instance, you can use the Federated Averaging strategy with a deep neural network or with a logistic regression but not with a random forest. Several FL strategies are already implemented in Substrafl.
```

Comment:
How can I add an internal link to the list of already implemented strategies https://connect-docs.owkin.com/en/latest/substrafl_doc/api/strategies.html ?

Reply:
it works thanks @Fabien-GELUS!